cmd/collector/main.go

```diff
@@ -15,26 +15,26 @@ import (
 )
 
 const (
-	ENV_COLLECT_INTERVAL_SECONDS = "COLLECTION_INTERVAL_SECONDS"
-	ENV_REPORT_INTERVAL_SECONDS  = "REPORT_INTERVAL_SECONDS"
-	ENV_MAX_ARCHIVE_AGE_DAYS     = "MAX_ARCHIVE_AGE"
+	collectingIntervalSeconds = "COLLECTION_INTERVAL_SECONDS"
+	reportIntervalSeconds     = "REPORT_INTERVAL_SECONDS"
+	maxArchiveAgeDays         = "MAX_ARCHIVE_AGE"
 
-	COLLECT_INTERVAL_DEFAULT_SECONDS = 300
-	REPORT_INTERVAL_DEFAULT_SECONDS  = 30
-	MAX_ARCHIVE_AGE_DEFAULT_DAYS     = 30
+	defaultCollectIntervalSeconds = 300
+	defaultReportIntervalSeconds  = 30
+	defaultMaxArchiveAgeDays      = 30
 )
 
 type options struct {
 	collectionInterval int
 	reportingInterval  int
-	maxArchiveAgeDays  int
+	maxArchiveAgeDays  time.Duration
```

Comment:
I suppose it could but I still feel including days in the variable name makes it clear what the duration is vs having to check in the `gatherOptions` func to see what the time period being discussed is. ie. the duration is still a # of days even if its converted to a duration before being added to the struct.

Reply:
OK, I've updated the var name.